src/Immunisatie-0-1/_components/variable.immunization.medication.url.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<nts:component xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript">
```

Comment:
Ik mis hier wel een stukje documentatie over welke parameters in de component zitten

Reply:
is toegevoegd